diagnostics/mixed_layer_depth/settings.jsonc

```diff
@@ -0,0 +1,38 @@
+// mixed_layer_depth MDTF diagnostic settings file
+//
+{
+  "settings" : {
+    "driver" : "mixed_layer_depth.py",
+    "long_name" : "Mixed Layer Depth Calculation",
+    "realm" : "ocean",
+    "description" : "Compute Mixed Layer Depth Mean from 3D Ocean Variables",
+    "pod_env_vars" : {
+      "EXAMPLE_FAV_COLOR": "blue",
+      "EXAMPLE_NEXT_FAV_COLOR": "red"
+    },
+    "runtime_requirements": {
+        "python3": ["scipy","matplotlib","xarray","netCDF4","xesmf","dask","numpy","pandas","cartopy","gsw"]
+    }
+  },
+
+  // In this section, you describe what model data your diagnostic uses. Add one
+  // list entry (the section in curly braces) for each variable, separated by 
+  // commas.
+  "varlist" : [
+    {
+      // Variable identifier. An environment variable with this name will be set
+      // to what the corresponding field is called in the model being analyzed.
+      // For example, if this diagnostic were run on CESM2, the variable $tas_var
+      // would be set to "TREFHT".
+      "var_name" : "so_var",
+      "freq" : "mon",
+      "requirement" : "required"
+    }
+    // If we needed a second variable, we'd list it here.
+   // {
+   //   "var_name" : "thetao_var",
```

Comment:
The main script reads in data for `thetao`, so this entry should be uncommented and added as a second required variable.

Reply:
When I try this, something breaks. Error message below. I guess something is wrong in the settings.jsonc syntax, but I can't see it

`Starting /glade/work/lettier/mdtf/MDTF-diagnostics/src/mdtf.py

Traceback (most recent call last):
  File "/glade/u/home/lettier/anaconda3/envs/mdtf/_MDTF_base/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/glade/u/home/lettier/anaconda3/envs/mdtf/_MDTF_base/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/glade/work/lettier/mdtf/MDTF-diagnostics/src/mdtf.py", line 321, in <module>
    mdtf.main_loop()
  File "/glade/work/lettier/mdtf/MDTF-diagnostics/src/mdtf.py", line 287, in main_loop
    case.pods.append(pod)
UnboundLocalError: local variable 'pod' referenced before assignment`